### PR TITLE
Fixes #900 - Removed obsolete attributes from skstat.

### LINKED ISSUE #900: Remove abandoned attributes from skstat and fix the man page
Several attributes related to link-routing and multi-tenancy have been left in `skstat`.  These should be removed and the man page for skstat should be brought up to date.

### PR DESCRIPTION
**Pull request by ted-ross**
Updated skstat manpage.
Closes #329

### CODE REVIEW

File: docs/man/skstat.8.adoc [hunk 1]
```diff
@@ -144,9 +145,14 @@ role::
...
   - 'normal' - The normal connection from a client to a router.
   - 'inter-router' - The connection between routers to form a network.
-  - 'route-container' - The connection to or from a broker or other host to receive link routes and waypoints.
+  - 'inter-router-data' - Member of a trunk of inter-router connections.  Used to carry streaming deliveries.
+  - 'route-container' - The connection to or from a broker or other host that has auto-links.
+  - 'inter-edge' - The connection between edge routers in an edge-mesh.
   - 'edge' - The connection between an edge router and an interior router.
 
+proto::
+The protocol being used over the connection.
```
Comment (pwright) [inline docs/man/skstat.8.adoc:154]:
```suggestion
The protocol being used over the connection, for example amqp or tcp
```

File: docs/man/skstat.8.adoc [hunk 2]
```diff
@@ -283,6 +280,9 @@ If this router is not a neighbor, this field identifies the next-hop neighbor us
 link::
 The ID of the link to the neighbor router.
 
+ver::
+The inter-router protocol version used by the router.
```
Comment (pwright) [inline docs/man/skstat.8.adoc:284]: , for example AMQP 1.0?

didn't see this in output:

```
kubectl exec --stdin --tty deploy/skupper-router -- skstat -n

2023-01-11 11:18:15.425186 UTC
east-skupper-router-6c8866fc67-nc448

Routers in the Network
Last Topology Change: Wednesday Jan 11 10:49:07 2023 GMT
  router-id                             next-hop  link
  ======================================================
  east-skupper-router-6c8866fc67-nc448  (self)    -
  west-skupper-router-79664f4686-qbmnx  -         0

```

File: docs/man/skstat.8.adoc [hunk 3]
```diff
@@ -315,8 +315,8 @@ The security or encryption method, if any, used for this connection.
 authentication::
 The authentication method and user ID of the connection's authenticated user.
 
-tenant::
-If the connection is to a listener using multi-tenancy, this column displays the tenant namespace for the connection.
+active::
+Whether this edge connection is active or standby.
```
Comment (pwright) [inline docs/man/skstat.8.adoc:319]: how do i get this in output:

```
kubectl exec --stdin --tty deploy/skupper-router -- skstat -e

Defaulted container "router" out of: router, config-sync
2023-01-11 11:45:40.052263 UTC
east-skupper-router-6c8866fc67-nc448
```

**Review by pwright — COMMENTED** [review]
Just questions ;)

File: docs/man/skstat.8.adoc (hunk 2 above)
Comment (ted-ross) [reply docs/man/skstat.8.adoc:284]: It appears if you use `skstat -nv`.

File: docs/man/skstat.8.adoc (hunk 3 above)
Comment (ted-ross) [reply docs/man/skstat.8.adoc:319]: To see any of the columns, you would have to have at least one edge router connected to the router you are querying.

File: docs/man/skstat.8.adoc (hunk 2 above)
Comment (ted-ross) [reply docs/man/skstat.8.adoc:284]: The version does not refer to `AMQP 1.0`.  It refers to the skupper-router-specific inter-router protocol used to discover topology and share information about address destinations.

File: docs/man/skstat.8.adoc [hunk 4]
```diff
@@ -144,9 +145,14 @@ role::
 The connection's role:
   - 'normal' - The normal connection from a client to a router.
   - 'inter-router' - The connection between routers to form a network.
-  - 'route-container' - The connection to or from a broker or other host to receive link routes and waypoints.
+  - 'inter-router-data' - Member of a trunk of inter-router connections.  Used to carry streaming deliveries.
```
[the remark this answers is not in the document]
Comment (ted-ross) [reply docs/man/skstat.8.adoc:148]: Agreed.  This needs to be clarified.  Maybe "A connection that runs in parallel with an inter-router connection to share the traffic load".

File: docs/man/skstat.8.adoc [hunk 5]
```diff
@@ -144,9 +145,14 @@ role::
...
+  - 'route-container' - The connection to or from a broker or other host that has auto-links.
+  - 'inter-edge' - The connection between edge routers in an edge-mesh.
```
[the remark this answers is not in the document]
Comment (ted-ross) [reply docs/man/skstat.8.adoc:150]: I think the original is more correct.  If you create a mesh of interior routers, the connections will be 'inter-router'.

File: docs/man/skstat.8.adoc (hunk 1 above)
Comment (ted-ross) [reply docs/man/skstat.8.adoc:154]: That's a good suggestion.

**Review by pwright — APPROVED** [review]

**Pull request merged**